### PR TITLE
Data Governance KIT Documentation

### PR DESCRIPTION
**Pull request by dg-tsy**
## Pre-review checks

Please ensure to do as many of the following checks as possible, before asking for committer review:

- [ ] DEPENDENCIES are up-to-date. [Dash license tool](https://github.com/eclipse/dash-licenses). Committers can open IP issues for restricted libs.
- [ ] [Copyright and license header](https://eclipse-tractusx.github.io/docs/release/trg-7/trg-7-02) are present on all affected files

### CODE REVIEW

**Comment by dg-tsy** [comment]
Hello Max,

I have created the GitHub issue for release checks - Data Governance Kit - Release Checks #542<https://github.com/eclipse-tractusx/sig-release/issues/542>

Kind regards / Viele Grüße
Deepak Gupta
Product Owner
Data Governance KIT
Catena-X

***@***.***

File: docs-kits/kits/Data Governance Kit/page_adoption-view.md [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+* All Data Consumers
+* Any Enablement Service Provider
+* Any Business Application Provider
+
+## NOTICE
+
+This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+* SPDX-License-Identifier: CC-BY-4.0
+* SPDX-FileCopyrightText: 2021,2022, 2023, 2024 T-Systems International GmbH
```
Comment (SebastianBezold) [inline docs-kits/kits/Data Governance Kit/page_adoption-view.md:89]: Are these dates correct? created in 2021?

File: docs-kits/kits/Data Governance Kit/resources/dgk-api-docs.yaml
Comment (SebastianBezold) [inline docs-kits/kits/Data Governance Kit/resources/dgk-api-docs.yaml:1]: Copyright header missing

**Review by SebastianBezold — COMMENTED** [review]
Hi @dg-tsy,

I think the copyright headers, that you added as comment to the `.md` files are unnecessary. We do have a Notice section for that. It does not hurt, but I think it is likely, that the Notice section and copyright header might not be updated consistently

File: docs-kits/kits/Data Governance Kit/page_adoption-view.md (hunk 1 above)
Comment (dg-tsy) [reply docs-kits/kits/Data Governance Kit/page_adoption-view.md]: Corrected.

File: docs-kits/kits/Data Governance Kit/page_changelog.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- First Version for Adoption View
+- First Version for Operation View
+- First Version for Development View
+- Notice
+
+### Changed
+
+### Removed
+
+<sub><sup>(C) 2021,2023 Contributors to the Eclipse Foundation. SPDX-License-Identifier: CC-BY-4.0</sup></sub>
```
[the remark this answers is not in the document]
Comment (dg-tsy) [reply docs-kits/kits/Data Governance Kit/page_changelog.md]: Corrected.

File: docs-kits/kits/Data Governance Kit/resources/dgk-api-docs.yaml
Comment (dg-tsy) [reply docs-kits/kits/Data Governance Kit/resources/dgk-api-docs.yaml:1]: This is a YAML file and cannot have any comments or anything else other than data.
I have also renamed this file to openAPI-yaml

**Comment by dg-tsy** [comment]
@maximilianong Here is the reply to your feedback comments:
* sidebar navigation for Data Governance Kit added.
* I could not find "Certificate Management Kit" anywhere. Could you please guide me to its correct path?
* Changelog is now updated.
* I thought that "Oparation View" was mandatory. When it's not mandatory, I can remove it in next iteration. 
  There will be no reference implementation.
* If I rename the development view to architecture view, then would I have an architecture view in addition to development view? If not, then where would the API specifications come? I propose to either keep it as it is or add two separate views architecture and development view. Please suggest.

I am still struggling about how to make "kits/Data Governance Kit/resources/openAPI.yaml" as a Tractus-X SwaggerHub API documentation. Any help in this regard is much appreciated.

**Comment by maximilianong** [comment]
Please fix the requested changes by 04.03.2024 (EOB) so that we can perform the final checks and prepare the release communication.

@dg-tsy we sent out the following explanation - please make sure you are on the Eclipse mailing list:

For APIs with reference implementation: 
 
Please use (or adapt existing) automated workflows to publish on SwaggerHub. Maintain your openAPI.yaml and the publishing workflow in the repository of the reference implementation. Remove everything related to that from the website repo and only link to SwaggerHub in your KIT from now on.
We utilize a user provided by the Eclipse Foundation to publish on SwaggerHub.
Credentials for this user can be used through the eclipse-tractusx GitHub org secrets and can be utilized in automated workflows. 
Pay special attention to the "swaggerHub" CLI commands.
The location where the openAPI file is maintained may vary. The BPDM example showcases how the application is initiated to download and publish the current file from that location.


      SWAGGERHUB_API_KEY: ${{ secrets.SWAGGERHUB_API_KEY }}
      SWAGGERHUB_USER: ${{ secrets.SWAGGERHUB_USER }}
 
 
Examples:
BPDM Certificate Management: [https://github.com/eclipse-tractusx/bpdm-certificate-management/blob/main/.github/workflows/publish-swagger-hub.yaml]

**Comment by maximilianong** [comment]
Operation view:  not mandatory if you do not have something to operate :D please remove it with this PR.

Regarding your suggestion for development view - its ok to go with your approach.

**Comment by danielmiehle** [comment]
@dg-tsy can you also fix your current linting issues, please?

image

**Comment by dg-tsy** [comment]
@maximilianong Following tasks completed:
- Operations View removed, also removed its references from /sidebarsDocsKits.js
- Linting issues resolved
- Added .mdx files for API details, hence changes suggested in [this comment](https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/701#issuecomment-1968565806) should not be required.

Please let me know if anything else is required. Looks like all changes done from my side.

**Review by maximilianong — APPROVED** [review]

**Review by danielmiehle — APPROVED** [review]

**Pull request merged**